src/tpm.rs

```diff
@@ -83,3 +83,27 @@ pub(crate) fn create_ek(
 
     Ok((handle, cert, tpm_pub_vec))
 }
+
+/* Retrieves an existing ek_handle from keylime.conf.
+ *
+ * Input: None
+ * Return: Key handle
+ *
+ * Example call:
+ * let ek_handle = tpm::get_ek();
+ */
+
+pub(crate) fn get_ek() -> Result<KeyHandle> {
+    let cfg_handle =
+        config_get("/etc/keylime.conf", "cloud_agent", "ek_handle")?;
```

Comment:
Perhaps have the function take an `&str` of the config value, and use that? Basically, I think it would be nice if this just did the parsing into a `KeyHandle`, and let the config handling over to the main functions.

Reply:
@puiterwijk Let me know if this shorter version is more like what you're talking about here. I can revise if not.